### PR TITLE
WEB-45, WEB-46: Employee fix and map fix

### PR DESCRIPTION
**Pull request by Miqqy**
…anged

### CODE REVIEW

File: pages/orders/[id].vue [hunk 1]
```diff
@@ -104,7 +104,7 @@ const activities = [
                 </el-timeline-item>
               </el-timeline>
             </div>
-            <div class="w-full h-60 pl-4 flex flex-col items-center justify-center">
+            <div class="w-full h-60 pl-4 flex flex-col items-center justify-center z-1">
```
Comment (NoahCardoza) [inline pages/orders/[id].vue:107]: Are we sure this appears above the button too?

File: components/NavigationBar.vue [hunk 2]
```diff
@@ -28,7 +28,7 @@ function navigateToSearch() {
     <template v-else>
       <el-menu-item index="/account">Account</el-menu-item>
       <el-menu-item index="/orders">Orders</el-menu-item>
-      <el-menu-item v-if="userStore.user.is_employee || userStore.user.is_superuser" index="/employee">Employee</el-menu-item>
+      <el-menu-item v-if="userStore.user.is_employee || userStore.user.is_superuser" index="/employee">Employee/Stock</el-menu-item>
```
Comment (NoahCardoza) [inline components/NavigationBar.vue]: Let's change this to go to `/employee/stock` and just title it "Stock" to the user.

**Review by NoahCardoza — CHANGES_REQUESTED** [review]
Lookie lookie por favor!

File: pages/orders/[id].vue (hunk 1 above)
Comment (Miqqy) [reply pages/orders/[id].vue:107]: The buttons to zoom in and out are still visible if that's what you mean

**Review by Miqqy — COMMENTED** [review]
fixed it good sir

**Review by NoahCardoza — APPROVED** [review]
LGTM!

**Pull request merged**